instances/etcd/variables.tf

```diff
@@ -1,28 +1,10 @@
 variable "network_cidrs" {
   type = "map"
```

Comment:
Why was this removed?

Reply:
Because it is defined in <root>/variables.tf as a input variable, and should be passed here.
```
network_cidrs                       | See map in variables.tf | A CIDR notation IP range of the VCN and its subnets.
```

`network_cidrs` is shared by `instances-etcd` and `instances-k8smaster`.